providers/openshift/provider.go

```diff
@@ -568,6 +568,44 @@ func (p *OpenShiftProvider) GetRedeemURL() (*url.URL, error) {
 	return redeemURL, err
 }
 
+func (p *OpenShiftProvider) DeleteSessionToken(session *providers.SessionState) error {
+	if len(session.AccessToken) == 0 {
+		return nil
+	}
+
+	// get an api path to the token
+	tokenAPIPath := getKubeAPIURLWithPath(fmt.Sprintf("/apis/oauth.openshift.io/v1/oauthaccesstokens/%s", session.AccessToken))
```

Comment:
We don't have a dep on openshift types or something?  I would have expected a "normal" client.